File: AzW.Web.Portal/src/models/RecoveryServiceVault.js
import ResourceType from './ResourceType'
export default class RecoveryServiceVault
{
    constructor (){
        this.GraphModel = {
            Id: '',
            ResourceType: ResourceType.RecoveryServiceVault(),
            DisplayName: ''
        };

        this.ProvisionContext = {
            ResourceType: ResourceType.RecoveryServiceVault(),
            Name: '',
            Location: '',
            ResourceGroupName: '',
            VMNamesToBackup: []

        };
        this.CalculatorContext = {
            IsFree: false,
            Tier: ''
        }
    }
}

File: Portal.Services/AzW.Model/Provision/RecoveryServiceVault.cs
namespace AzW.Model
 {
     public class RecoveryServiceVault : Resource
     {
         public string Name { get; set; }
         public string Location { get; set; }
         public string ResourceGroupName { get; set; }
         public string[] VMNamesToBackup { get; set; }
     }
 }